# tip0003

```shell

>>> data = User.objects.filter(last_login=None)
>>> data.get()
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/Users/a13400566/devcode/airtelafrica/env/CoreKMP/lib/python3.7/site-packages/django/db/models/query.py", line 442, in get
    num if not limit or num < limit else 'more than %s' % (limit - 1),
django.contrib.auth.models.User.MultipleObjectsReturned: get() returned more than one User -- it returned more than 20!
>>> data.count()
2672

---
2022-09-13T06:52:54.611513Z	   36 Query	SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined` FROM `auth_user` WHERE `auth_user`.`last_login` IS NULL LIMIT 21
2022-09-13T06:53:22.860311Z	   36 Query	SELECT COUNT(*) AS `__count` FROM `auth_user` WHERE `auth_user`.`last_login` IS NULL

```